src/components/Menu.vue

```diff
@@ -1,5 +1,18 @@
 <template>
   <div id="controls">
+    <span
+      class="nomlogSummary"
```

Comment:
Please use kebab-case for CSS class names. :smiley:

Reply:
willdo